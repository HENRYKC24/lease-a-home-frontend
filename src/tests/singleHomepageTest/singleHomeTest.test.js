import React from 'react';
import { Provider } from 'react-redux';
import { shallow, configure } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import store from '../../redux/configureStore';
import Detail from '../../components/singleHome/singleHome';

configure({ adapter: new Adapter() });

jest.mock('react-router-dom', () => ({
  useLocation: jest.fn().mockReturnValue({
    pathname: '/another-route',
    search: '',
    hash: '',
    state: null,
    key: '5nvxpbdafa',
  }),
}));

describe('App component', () => {
  it('Renders correctly', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <Detail />
      </Provider>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
