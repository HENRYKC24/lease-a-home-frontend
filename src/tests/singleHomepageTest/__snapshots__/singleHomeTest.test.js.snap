// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Renders correctly 1`] = `ShallowWrapper {}`;
